test/test_linalg.py

```diff
@@ -1638,7 +1638,12 @@ def run_test_case(input, p):
         a = torch.eye(3, dtype=dtype, device=device)
         a[-1, -1] = 0  # make 'a' singular
         for p in norm_types:
-            run_test_case(a, p)
+            try:
+                run_test_case(a, p)
+            except numpy.linalg.LinAlgError:
+                # Numpy may fail to converge for some BLAS backends (although this is very rare)
+                # See the discussion in #67675
```

Comment:
It's not clear whether it's referring to a PR or an issue and in which repository NumPy or PyTorch.
```suggestion
                # See the discussion in https://github.com/pytorch/pytorch/issues/67675
```